.github/workflows/publish_helm_chart.yml

```diff
@@ -42,6 +42,12 @@ jobs:
           mask-password: 'true'
           registry-type: public
 
+      - name: install helm with version greater than v3.13.1
+        uses: azure/setup-helm@v3
+        with:
+          version: "v3.9.2" # default is latest (stable)
```

Comment:
how is 3.9.2 greater than 3.13.1?

Reply:
ah yes, sorry. wrong comment. i used 3.13.1 but that didn't work so I then decided to go with my local version